packages/documentation/package.json

```diff
@@ -14,15 +14,14 @@
 	},
 	"description": "Kotti-ui Doc Pages",
 	"devDependencies": {
-		"@metatypes/typography": "^0.4.0",
-		"@metatypes/units": "^0.4.0",
+		"@metatypes/typography": "^0.5.0",
+		"@metatypes/units": "^0.5.0",
 		"@nuxt/types": "^2.13.2",
-		"@nuxt/typescript-build": "^1.0.3",
-		"@nuxt/typescript-runtime": "^0.4.10",
-		"@nuxtjs/axios": "^5.11.0",
-		"@nuxtjs/markdownit": "^1.2.7",
-		"@nuxtjs/markdownit-loader": "^1.1.1",
-		"@nuxtjs/style-resources": "^1.0.0",
+		"@nuxt/typescript-build": "2.x",
+		"@nuxt/typescript-runtime": "2.x",
```

Comment:
this package becomes obsolete from Nuxt 2.15 onwards but we can't upgrade to nuxt 2.15 without upgrading vue-loader to 2.7 (which is the naruto upgrade branch)